tests/Feature/QueryBuilder/ExceptionHandlingTest.php

```diff
@@ -27,7 +27,7 @@ public function the_ignore_strategy_will_ignore_the_error_and_continue()
         // 'name' filter expects array.
```

Comment:
This comment is no longer correct, since it now expects a string, not an array. Can you either remove of update it?

Reply:
Comment updated